docs/releasing-a-xpi.md

```diff
@@ -61,7 +61,8 @@ If we need a new release build:
 
 | Group name | Who | Type of xpi | Notes |
 | ---------- | --- | ----------- | ----- |
-Privileged webextension admin | product-delivery team (+releng as backups) | privileged | Two of these are required to sign off on `privileged` xpis.
+Privileged webextension admin | Add-on Review team | privileged | Two are required to sign off on `privileged` xpis, with one being a member of the Add-on Review team.
+Privileged normandy admin | releng | normandy-privileged | Two are required to sign off on `normandy-privileged` xpis.
```

Comment:
Both of these should probably read `Normandy review team (+releng as backups)`, unless we change that in https://github.com/mozilla-releng/shipit/pull/373 .

Reply:
What do you mean by "both"?

Reply:
I tried addressing this in f0d6920. Please let me know if there is something else you have in mind.

Reply:
By `both` I mean it would look like

```
Privileged normandy admin | Normandy review team (+releng as backups) | Normandy review team (+releng as backups) | Two are required to sign off on `normandy-privileged` xpis.
```

Reply:
This should probably read:

```
Add-on Review Team | Add-on Review team | privileged | One of each (Add-on Review Team and Privileged webension admin) is required to sign off on `privileged` xpis.
Privileged webextension admin | Add-on Review team (+releng as backups) | privileged | One of each (Add-on Review Team and Privileged webension admin) is required to sign off on `privileged` xpis.
Normandy privileged admin | Normandy review team (+releng as backups) | normandy-privileged | Two are required to sign off on `normandy-privileged` xpis. 
```

The `Group name` is supposed to match the text in the shipit UI, e.g. [here](https://github.com/mozilla-releng/shipit/pull/373/files#diff-dc169779ebfd03bbc83bd865c3ee9ffeR501).

If we want to make it a bit clearer, we could make the add-on review team the `admin` and the other `team`, to match mozillaonline's pattern. Not a blocker.

Reply:
> If we want to make it a bit clearer, we could make the add-on review team the admin and the other team, to match mozillaonline's pattern. Not a blocker.

Sorry, I am not sure what you are suggesting here.

Reply:
The [two mozillaonline signoff groups](https://github.com/mozilla-extensions/xpi-manifest/pull/67/files#diff-c81418c7017824299abe9eab36e90b74R68) are named `MozillaOnline privileged webextension team` and `MozillaOnline privileged webextension admin`. The latter is a subset of the former. Because they both have similar names but are differentiated by `team` versus `admin`, it's hopefully clear that they are mozillaonline-specific teams, but one is for the team, and the other is for the administrators.

We could rename `Add-on Review Team` and `Privileged webextension admin` to match that pattern, but I'm ok if we land without that change. If we do rename, we'll need to update both the docs and the shipit UI strings.

Reply:
Ah, I see what you mean. I'd like to keep "Add-on Review Team" to make it clear to team members and others which one they should choose.
If you prefer, I could rename them to "Add-on review admin" and "Privileged webextension team".

Reply:
Looks like Rail landed the shipit code, so I think we can land this as-is :)

Reply:
Ok, let me just push the doc update then.

Reply:
Oh, I already have :)